src/components/VoiceCompair/VoiceCompair.js

```diff
@@ -211,8 +213,34 @@ const VoiceCompair = props => {
 
         let word_result = (result_per_words == 100) ? "correct" : "incorrect";
 
+        const client = new S3Client({
```

Comment:
can we add this in one place? looks like it duplicated in - both components

Reply:
Added in config file